packages/gatsby/src/bootstrap/get-plugin-hash.js

```diff
@@ -0,0 +1,40 @@
+const md5File = require(`md5-file/promise`)
+const path = require(`path`)
+const _ = require(`lodash`)
+
+module.exports = async function getPluginHash({
```

Comment:
the same function name as in `packages/gatsby/src/bootstrap/cache/plugin-hash.js`


```suggestion
module.exports = async function getPluginHash({
```

Reply:
this was generally unused so I removed the file completely